xml-validator/setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup
+
+setup(
+    name='xml-validator',
+    version='0.1',
+    py_modules=['validator'],
+    install_requires=[
+        'Click',
+        'xmlschema',
+    ],
+    entry_points='''
+        [console_scripts]
+        xml-validator=validator:cli
```

Comment:
```suggestion
        xml-validator=xml-validator:validator:cli
```

move `setup.py` to root directory and the entry point will look like above, however might be a good idea to rename the entry point to not be the same name as the folder of the python package